package.json

```diff
@@ -10,10 +10,11 @@
     "url": "git+https://github.com/arition/torch-js.git"
   },
   "scripts": {
-    "install": "yarn compile",
+    "install": "yarn clean && yarn compile",
     "build": "node build.js",
     "compile": "cmake-js compile",
-    "rebuild": "cmake-js rebuild"
+    "rebuild": "cmake-js rebuild",
+    "clean": "rm -rf libtorch-cpu.zip"
```

Comment:
I don't think we need the clean scripts here. For other platforms, the downloaded zip may have a different name. The cmake-js will also automatically redownload the zip if the build process failed.

Reply:
Fair enough. I had it on there because it wasn't auto-redownloading on MacOS - that's something that we should probably look into individually though.